doc/reconstruction_models_list.rst

```diff
@@ -0,0 +1,222 @@
+        <td><a href='https://www.sciencedirect.com/science/article/pii/S1053811911006148'>Fieremans 2011</a></td>
+    </tr>
+    <tr>
+        <td><a href='#diffusion-kurtosis-imaging'>Mean Signal DKI</a></td>
+        <td class='red'>No</td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td><a href='https://www.repository.cam.ac.uk/handle/1810/281993'>Henriques, 2018</a></td>
+    </tr>
+    <tr>
+        <td><a href='#q-ball-constant-solid-angle'>CSA</a></td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td><a href='https://www.ncbi.nlm.nih.gov/pubmed/20535807'>Aganj 2010</a></td>
+    </tr>
+    <tr>
+        <td>Westin's CSA</td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td><a href='#intravoxel-incoherent-motion-ivim'>IVIM</a></td>
+        <td class='red'>No</td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td><a href=''>LeBihan 1984</a></td>
+    </tr>
+    <tr>
+        <td><a href='#intravoxel-incoherent-motion-ivim'>IVIM Variable Projection</a></td>
+        <td class='red'>No</td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td><a href=''>Fadnavis 2019</a></td>
+    </tr>
+    <tr>
+        <td>SDT</td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td><a href=''>Descoteaux 2009</a></td>
+    </tr>
+    <tr>
+        <td><a href='#diffusion-spectrum-imaging'>DSI</a></td>
+        <td class='red'>No</td>
+        <td class='red'>No</td>
+        <td class='green'>Yes</td>
+        <td></td>
+        <td><a href=''>Wedeen 2008</a></td>
+    </tr>
+    <tr>
+        <td><a href='#dsi-with-deconvolution'>DSID</a></td>
+        <td class='red'>No</td>
+        <td class='red'>No</td>
+        <td class='green'>Yes</td>
+        <td></td>
+        <td><a href=''>Canales-Rodriguez 2010</a></td>
+    </tr>
+    <tr>
+        <td><a href='#generalized-q-sampling-imaging'>GQI - GQI2</a></td>
+        <td class='red'>No</td>
+        <td class='green'>Yes</td>
+        <td class='green'>Yes</td>
+        <td></td>
+        <td><a href=''>Yeh 2010</a></td>
+    </tr>
+    <tr>
+        <td><a href='#sparse-fascicle-model'>SFM</a></td>
+        <td class='green'>Yes</td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td><a href=''>Rokem 2015</a></td>
+    </tr>
+    <tr>
+        <td><a href='#q-ball-constant-solid-angle'>Q-Ball (OPDT)</a></td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td><a href=''>Tuch 2004</a>, <a href=''>Tristan-Vega 2010</a></td>
```

Comment:
I think you should also have the Descoteaux reference:

` Regularized, Fast, and Robust Analytical Q-Ball Imaging. M. Descoteaux, E. Angelino, S. Fitzgibbons, and R. Deriche. Magnetic Resonance in Medicine, 58 (3) , 2007`